source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java

```diff
@@ -804,6 +804,87 @@ private String saveProject() {
                 } catch (ParserConfigurationException e) {
                     e.printStackTrace();
 
+        EditText authorEditText = (EditText) findViewById(R.id.author_name);
```

Comment:
@octacode 
Why you make changes here?
Travis is failing, check the log here : https://travis-ci.org/BuildmLearn/BuildmLearn-Toolkit-Android/builds/211018551